src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -49,3 +34,27 @@
     (utils/get-account-by-address accounts account-address)
     :balance
     (utils/get-balance-by-address balances account-address))))
+
+(defn- prepare-token
+  [{:keys [symbol marketValuesPerCurrency] :as item}]
+  (let [fiat-value                      (utils/sum-token-chains item)
+        market-values                   (:usd marketValuesPerCurrency)
+        {:keys [price changePct24hour]} market-values
+        fiat-change                     (* fiat-value (/ changePct24hour (+ 100 changePct24hour)))]
+    {:token               (keyword (string/lower-case symbol))
+     :state               :default
+     :status              (cond
+                            (pos? changePct24hour) :positive
+                            (neg? changePct24hour) :negative
+                            :else                  :empty)
+     :customization-color :blue
+     :values              {:crypto-value      (.toFixed (* fiat-value price) 2)
+                           :fiat-value        (utils/prettify-balance fiat-value)
+                           :percentage-change (.toFixed changePct24hour 2)
+                           :fiat-change       (utils/prettify-balance fiat-change)}}))
+
+(re-frame/reg-sub
+ :wallet/parsed-tokens
```

Comment:
`:wallet/parsed-tokens` doesn't describe what exactly subscription returns. I'd suggest renaming it to something like `:wallet/actual-token-values` or something like that